index.js

```diff
@@ -721,7 +723,7 @@ class KeyringController extends EventEmitter {
    */
   setUnlocked () {
     this.memStore.updateState({ isUnlocked: true })
-    this.emit('unlock', true)
+    this.emit('unlock')
```

Comment:
This is a breaking change, no? It's unfortunate but someone could be using this for something.